lib/src/liveapp.ts

```diff
@@ -7,4 +7,5 @@ export const QueryParams = {
   FromAccountId: "fromAccountId",
   ToAccountId: "toAccountId",
   FeeStrategy: "feeStrategy",
+  Rate: "rate",
```

Comment:
The rate shouldn't be provide by the LiveApp itself?
This is a provider responsibility.

Reply:
As we use "float" rate, I find confused for the user that we provide different "rates" across the flow... up to us
(I can move it to a non query params for swap-live, if we decide change the rate in the middle of the flow).

Reply:
Following this conversation the param has been moved to InternalParams